tests/raftstore/cluster.rs

```diff
@@ -163,10 +167,11 @@ impl<T: Simulator> Cluster<T> {
     }
 
     pub fn call_command(&self,
+                        node_id: u64,
```

Comment:
Yes, but the condition *store_id the message wants to be sent is equal to the store_id in header* still be true.